### PR TITLE
Improve startup performance and reduce runtime GC

### PR DESCRIPTION
**Pull request by NogginBops**
### The issue this fixes

The code is doing a lot of unnecessary string concatenations.

### What this PR does

This PR changes how the SpriteManager stores sprites to avoid string concatenations.

### What more can be done

There is a possibility that tile initialization on startup doesn't need to do all the things it is doing which could lead to further speedups.

### CODE REVIEW

**Comment by NogginBops** [comment]
Also removes "ph_" prefix for sprites as the complexity seems unnecessary (and adds GC).
People can submit PRs for any sprite they wish to change.

**Comment by BraedonWooding** [comment]
We’ll need to run some performance tests before/after to see the difference it makes

File: Assets/Scripts/Controllers/Sprites/FurnitureSpriteController.cs [hunk 1]
```diff
@@ -324,6 +335,7 @@ private string GetSuffixForDiagonalNeighbour(string suffix, string coord1, strin
     {
         if (suffix.Contains(coord1) && suffix.Contains(coord2))
         {
+            // FIXME: Doing ToLower here sucks!
```
Comment (BraedonWooding) [inline Assets/Scripts/Controllers/Sprites/FurnitureSpriteController.cs]: Coordinates can only be N,E,S,W so lookup table if it really is a perf concern could also act as a validation measure.

**Comment by BraedonWooding** [comment]
You seem to have done a merge which basically means the last few commits are also reflected here.  This needs to be cleaned up a little so it is much more clear what changes you have made in this PR.

**Pull request merged**